staging/src/github.com/kubeedge/mapper-framework/_template/mapper/Dockerfile

```diff
@@ -7,6 +7,8 @@ ENV GO111MODULE=on \
 
 COPY . .
 
+RUN  go mod tidy
```

Comment:
It doesn't need to be declared

Reply:
You can refer to the issue #5287 I raised. Since the _template file seems to be skipped when executing `make verify` to detect dependencies, the dependency on the database cannot be added to go.mod, resulting in an error during docker build, so I executed `go mod tidy` here to refresh go.mod after generating the mapper project. @WillardHu

Reply:
It really depends on the user generated project, and you can execute go mod tidy in the generated project.

Reply:
Do you mean that we execute `go mod tidy` directly when generating the user mapper project instead of when building the image?

Reply:
Yes, this is more suitable for the user's use scenario, and the user local debug will actually update the go.mod

Reply:
After testing, it can indeed achieve the same effect. I have re-edited